test/boost/sstable_3_x_test.cc

```diff
@@ -3382,9 +3382,8 @@ static void write_mut_and_validate_version(test_env& env, schema_ptr s, const ss
     lw_shared_ptr<memtable> mt = make_lw_shared<memtable>(s);
     mt->apply(mut);
 
-    tmpdir tmp = write_and_compare_sstables(env, s, mt, table_name, version);
+    tmpdir tmp = write_sstables(env, s, mt, version);
     auto written_sst = validate_read(env, s, tmp.path(), {mut}, version);
-    do_validate_stats_metadata(s, written_sst, table_name);
```

Comment:
Why this?

Reply:
it compares metadata with the reference.  if the amount of range tombstone entries is different (due to splits), the metadata validation fails.

Reply:
Lets add a FIXME to remind us to restore or at least revisit this after the v2 migration is done.